<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>599</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>599</width>
    <height>485</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>NotasExpress</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logo/icons/iconProgram.ico</normaloff>:/logo/icons/iconProgram.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
    font-family: 'Segoe UI', sans-serif;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #198754;
font-weight: bold;
color: white;
border: none;
border-radius: 4px;
</string>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/icons/save_pdf.png</normaloff>../resources/icons/save_pdf.png</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>11</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Reddit Sans Chocolate</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333333;
padding: 3px;</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Reddit Sans Chocolate'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;ESCUELA TÉCNICA INDUSTRIAL NACIONAL GIOVANNI FERRAREIS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>81</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>ARCHIVO:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathFile">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
padding: 3px;
border: 1px solid #cccccc;
border-radius: 3px;
color: #333333;
}

QLineEdit:focus {
    border-color: #2A3663;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SelectButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #2A3663;
text-decoration: none;
color: white;
border: none;
font-weight: bold;
padding: 5px 10px;
border-radius: 4px;
margin: 3px;</string>
             </property>
             <property name="text">
              <string>Seleccionar</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resources/icons/select.png</normaloff>../resources/icons/select.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>DESTINO:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathFolder">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
padding: 3px;
border: 1px solid #cccccc;
border-radius: 3px;
color: #333333;
}

QLineEdit:focus {
    border-color: #2A3663;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="SaveButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #2A3663;
text-decoration: none;
color: white;
border: none;
font-weight: bold;
padding: 5px 10px;
border-radius: 4px;
margin: 3px;</string>
           </property>
           <property name="text">
            <string>Guardar en...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/icons/save.png</normaloff>../resources/icons/save.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>43</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>PROFESOR GUÍA:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="GuideTeacherEntry">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
padding: 3px;
border: 1px solid #cccccc;
border-radius: 3px;
color: #333333;
}

QLineEdit:focus {
    border-color: #2A3663;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>43</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>FECHA DE ENTREGA:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DateEntry">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
padding: 3px;
border: 1px solid #cccccc;
border-radius: 3px;
color: #333333;
}

QLineEdit:focus {
    border-color: #2A3663;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>AÑO - SECCIÓN:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CbYearSection">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">/* Estilos para los combobox */
QComboBox {
    padding: 3px;
    border: 1px solid #cccccc;
    border-radius: 4px;
    color: #333333;
    selection-background-color: #2A3663; 
    selection-color: white;
}

QComboBox:hover {
    border-color: #2A3663; 
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left-width: 1px;
    border-left-color: #cccccc;
    border-left-style: solid;
    border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
    background-color: #f0f0f0;
}

QComboBox::down-arrow {
    width: 12px;
    height: 12px;
}

QComboBox::down-arrow:on { /* Cuando el combobox está abierto */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
    selection-background-color: #2A3663; 
    selection-color: white;
    padding: 4px;
    outline: 0; /* Elimina el borde de enfoque predeterminado */
}

QComboBox QAbstractItemView::item {
    padding: 6px;
    border-radius: 2px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #e0e0e0; /* Gris claro al pasar el mouse */
}

QComboBox QAbstractItemView::item:selected {
    background-color: #2A3663; 
    color: white;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Reddit Sans Chocolate</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #333333;
padding: 3px;</string>
             </property>
             <property name="text">
              <string>MENCIÓN:</string>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CbMention">
             <property name="sizeIncrement">
              <size>
               <width>600</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">/* Estilos para los combobox */
QComboBox {
    padding: 3px;
    border: 1px solid #cccccc;
    border-radius: 4px;
    color: #333333;
    selection-background-color: #2A3663; 
    selection-color: white;
}

QComboBox:hover {
    border-color: #2A3663; 
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left-width: 1px;
    border-left-color: #cccccc;
    border-left-style: solid;
    border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
    background-color: #f0f0f0;
}

QComboBox::down-arrow {
    image: url(:/icons/down_arrow.png);  
    width: 12px;
    height: 12px;
}

QComboBox::down-arrow:on { /* Cuando el combobox está abierto */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
    selection-background-color: #2A3663; 
    selection-color: white;
    padding: 4px;
    outline: 0; /* Elimina el borde de enfoque predeterminado */
}

QComboBox QAbstractItemView::item {
    padding: 6px;
    border-radius: 2px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #e0e0e0; /* Gris claro al pasar el mouse */
}

QComboBox QAbstractItemView::item:selected {
    background-color: #2A3663; 
    color: white;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="Table">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">/* Estilos para la tabla */
QTableWidget {
    background-color: white;
    border: 1px solid #cccccc;
    border-radius: 3px;
    gridline-color: #cccccc;
}

QTableWidget::item {
    padding: 3px;
}

QTableWidget::item:selected {
    background-color: #4CAF50; /* Verde para la selección */
    color: white;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>95</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>N°</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CÉDULAS</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>NOMBRES</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>APELLIDOS</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PROMEDIO</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/logoe.qrc"/>
 </resources>
 <connections/>
</ui>
